web-src/src/pages/PageComposerTracks.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
+    <tabs-music></tabs-music>
```

Comment:
We don't show the tabs on e. g. artist page. Please drop it to be consistent with the other pages.